### DISCUSSION TITLE
Ubuntu is not supported - by design

### DISCUSSION THREAD

**Discussion by ghost** [telekom-security/tpotce#1134] [General]
AWS  Ubuntu 22.04 EC2. Install fails with "Aborting. Debian jammy is not supported." using commands below.

git clone https://github.com/telekom-security/tpotce
cd tpotce/iso/installer/
cp tpot.conf.dist tpot.conf
./install.sh --type=auto --conf=tpot.conf

This looks like a really cool app but seems clearly broken for 2022 recent versions of Ubuntu (jammy etc..). anyone got any idea how to make this run on Ubuntu 22.04 AWS EC2?

**Comment by t3chn0m4g3** [comment]
People just do not read the documentation anymore or they would know that Ubuntu is not supported.

- I adjusted the topic accordingly.

**Reply by ghost** [reply]
Thank you. In my defense I read the page in its entirety although it was 2 am and I was sleepy. My bad on that one! I did try Debian last night however and it worked like a charm. Very nice! Amazing work.

**Reply by t3chn0m4g3** [reply]
Thanks, we have all been there 😄
